### ISSUE TITLE
A11Y: Side navigation is focusable when off-screen

### ISSUE THREAD

**Issue by bartaz** [canonical-web-and-design/vanilla-framework#4134]
Side navigation (in docs layout) on small screen gets moved off screen and can be toggled back to view.
But even off screen when using tab to move focus, or screen reader, it is accessible and may be confusing.

We should make sure it's either completely hidden (not focusable), or gets into view when focused via keyboard.
